### DISCUSSION TITLE
More visible loading-indicator for logs

### DISCUSSION THREAD

**Discussion by gabor** [grafana/grafana#50598] [Observability]
it seems some users do not spot the moving-blue-line at the top of the logs-section, that indicates that the logs are being loaded. maybe we should improve it to be more visible.

(issue where it was mentioned: https://github.com/grafana/grafana/issues/49333)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
